pallets/inflation/src/lib.rs

```diff
@@ -289,7 +287,7 @@ pub mod pallet {
         ///
         /// Purpose of the call is testing & handling unforeseen circumstances.
         #[pallet::call_index(1)]
-        #[pallet::weight(T::WeightInfo::force_inflation_recalculation())]
+        #[pallet::weight(T::WeightInfo::force_inflation_recalculation().saturating_add(T::DbWeight::get().writes(1)))]
```

Comment:
Asking out of curiosity, why this additional one write weight is now needed?

Reply:
Storage is whitelisted so it's not accounted for anywhere. Only reads are accounted for. It's a legacy mistake, not really important, and unrelated to this PR.